paf-mvp-cmp/src/view.ts

```diff
@@ -34,25 +32,21 @@ export class View implements IView {
   private outerContainer: HTMLElement = null;
 
   // The locale that the UI should adopt.
-  private readonly locale: Locale;
+  private readonly locale: ILocale;
 
   // The options provided to the controller.
   private readonly config: Config;
 
   /**
    * Constructs a new instance of View
    * @param script element this method is contained within
-   * @param locale the language file to use with the UI
+   * @param locale the language text to use with the UI
    * @param config the configuration for the controller
    */
-  constructor(script: HTMLOrSVGScriptElement, locale: Locale, config: Config) {
+  constructor(script: HTMLOrSVGScriptElement, locale: ILocale, config: Config) {
     this.script = script;
     this.config = config;
-    this.locale = locale;
-
-    // Setup the locale with the text and images to use.
-    this.locale.Logo = logoSvg;
-    this.locale.LogoCenter = logoCenterSvg;
+    this.locale = <ILocale>locale;
```

Comment:
[NIT] This casting is useless as the parameter is already typed in the constructor signature

Reply:
Resolved.